src/ExpressionEvaluator/VisualBasic/Test/ResultProvider/BasicResultProviderTest.vbproj

```diff
@@ -86,12 +86,6 @@
     <Compile Include="..\..\Source\ResultProvider\Helpers\TypeExtensions.vb">
```

Comment:
Consider adding `<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->` at the top of the file after the xml declaration.